### DISCUSSION TITLE
Getting ModuleNotFoundError in fast_stable_diffusion_AUTOMATIC1111.ipynb

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#1136] [General]
This notebook was working perfectly last night but when I tried to run it now, I am getting this error. I also downloaded the latest notebook, deleted every folder, run it again. Still the same error. First time posting here, so apologies in advance if I made any mistake.

![image](https://user-images.githubusercontent.com/76521534/209430780-3ed7099a-2ada-4dce-9823-66ba76f90108.png)

**Comment by TheLastBen** [comment]
fixed, remove the folder sd and do a clean run with the latest colab https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Reply by i-huzaifa-arshad** [reply]
/usr/local/lib/python3.8/dist-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.16.5 and <1.23.0 is required for this version of SciPy (detected version 1.24.0
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 195, in <module>
    webui()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 133, in webui
    initialize()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 63, in initialize
    modules.sd_models.load_model()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py", line 284, in load_model
    checkpoint_info = checkpoint_info or select_checkpoint()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py", line 115, in select_checkpoint
    if len(model_checkpoint) == 0:
TypeError: object of type 'NoneType' has no len()

**Reply by TheLastBen** [reply]
did you remove the folder sd ?

**Reply by i-huzaifa-arshad** [reply]
yes sir I did. I deleted each and every folder related to SD. Re run it, still same error

**Reply by TheLastBen** [reply]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/5971
